python/cudf/cudf/tests/test_series.py

```diff
@@ -1813,3 +1813,17 @@ def test_series_digitize_invalid_bins():
         ValueError, match="`bins` cannot contain null entries."
     ):
         _ = s.digitize(bins)
+
+
+@pytest.mark.parametrize("data", [[0, 1, 2, 3, 4, 5, 10]])
```

Comment:
We need to test NaN somewhere, since the docs specify that this always returns `False` for NaN.

Reply:
Added tests for this additional coverage.